src/workspace-data/data-table/shared/EntityUploader.js

```diff
@@ -49,8 +50,9 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
   const [recordTypeInputTouched, setRecordTypeInputTouched] = useState(false);
 
   // Google workspace regions are hardcoded for now, as GCP uploads to the Rawls service which is only on uscentral-1
-  const regionLabelToDisplay = isGoogleWorkspace ? 'US' : getRegionLabel(region);
-  const regionFlagToDisplay = isGoogleWorkspace ? '🇺🇸' : getRegionFlag(region);
+  const rawlsRegion = getRegionInfo('US-CENTRAL1', locationTypes.region);
```

Comment:
This seemed like an oversight that we were not showing central1, despite the comment at line 52.

Before:
<img width="591" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/484484/be0973fc-b461-4f97-95fd-4e4174390473">

After: 
<img width="589" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/484484/f0eb04c8-2435-4027-86f2-ce89366e6791">

Reply:
Perhaps this would be better reworded as "Data will be saved in the following Terra-managed location: us-central1 (Iowa)."?

Reply:
@jladieu did you have any opinion about me rewording this?

Reply:
After discussing...yes I like it :)

Reply:
Simplifying text as discussed in Slack with @jladieu. I dropped the period at the end (the dialog already has some precedence for that), but could add it back if anyone feels strongly.

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/bed6fe17-6271-42fc-9a67-4c22a74df3cc)